docs/tutorial/tutorial-5/android.rst

```diff
@@ -2,9 +2,159 @@
 Tutorial 5 - Taking it mobile: Android
 ======================================
 
-To deploy our applicaton to Android...
+The process of deploying an application to Android is very similar to the
+process for deploying as a desktop application. There is one additional
+pre-requisite: Java 8.
+
+Setting up Java 8
+=================
+
+To use BeeWare's Android tooling, you must install a Java 8 compiler and provide
+the path to it in the ``JAVA_HOME`` environment variable. This is because
+BeeWare's Android tooling depends on the Android SDK, and the Android SDK
+depends on the Java runtime and compiler being available. Some tools within
+the Android SDK specifically require version 8 of Java, so we stick to that
+in this documentation.
+
+Download & install Java 8
+-------------------------
+
+Visit the `AdoptOpenJDK website <https://adoptopenjdk.net/>`_ and
+download OpenJDK 8.
+
+On macOS, the default format is a *.pkg* installer. Download and execute the
+installer, accepting the defaults. In your terminal, run:
+``export JAVA_HOME="$(/usr/libexec/java_home -v 1.8)"``
+
+On Windows, the default is the *.msi* installer. Download and execute the
+installer. During the install process, look for an option labeled
+**Set JAVA_HOME variable**. Ensure it is enabled; if a red letter *X* precedes it,
+click the red letter *X* and choose **Will be installed on local hard drive**.
+
+On Linux, the default is a _.tar.gz_ file. Download it and unpack it to a
+directory of your choosing. Set the *JAVA_HOME* environment variable to the
+directory containing *bin*; for example, if you unpacked the *.tar.gz* file
+into ``~/java/1.8/``, at the time of writing, you would need to run
+``export JAVA_HOME=$HOME/java/1.8/jdk8u242-b08``.
+
+Testing your Java 8 environment
+-------------------------------
+
+To make sure that the Java compiler is available via the *JAVA_HOME*
+environment variable, and that it is version 8, run the following command.
+
+We test *javac* here, not *java*, because if *javac* works, so will *java*.
+
+If you get a "Command not found" error on macOS or Linux, look for the
+``export`` command in the previous section, and run it.
+
+.. tabs::
+
+  .. group-tab:: macOS
+
+    .. code-block:: bash
+
+      $ "$JAVA_HOME/bin/javac" -version
+      javac 1.8.0_242
+
+  .. group-tab:: Linux
+
+    .. code-block:: bash
+
+      $ "$JAVA_HOME/javac" -version
+      javac 1.8.0_242
+
+  .. group-tab:: Windows
+
+    .. code-block:: doscon
+
+      C:\...>"%JAVA_HOME%\bin\javac" -version
+      javac 1.8.0_242
+
+Create an Android app and compile it
+====================================
+
+The process of deploying an application to Android is very similar to the
+process for deploying as a desktop application. First, you run the ``create``
+command. This downloads an Android app template and adds your Python code to it.
+
+.. code-block:: bash
+
+  (beeware-venv) $ briefcase create android
```

Comment:
This should be in tabs as well. Even though the command and output is the same, we're using tabs to provide a distinct prompt. The duplication is annoying, but there's not much else we can do about it.